doc/api.rst

```diff
@@ -73,8 +68,10 @@ Stats utils
 .. autosummary::
     :toctree: generated/
 
+    autocov
     autocorr
     make_ufunc
+    wrap_xarray_ufunc
```

Comment:
Nor `make_ufunc` nor `wrap_xarray_ufunc` are in "stats/__init__.py", thus, the docs are not properly generated as they can not be imported.

Reply:
Thanks for catching this. I think they could be added to init, because they are quite flexible functions.

On the other hand, I think we can skip them now and let's re-evaluate this point after the summer.

Reply:
Same here, do whatever you feel like. I only commented it here and not in init because init was not modified.